### DISCUSSION TITLE
Trying To Use Reoccurring blocks but can't

### DISCUSSION THREAD

**Discussion by jackbat4** [SelfControlApp/selfcontrol#819] [Q&A]
I'm trying to use Andreas Grill's extension to set up a schedule for blocks - I'm sure that I've done the work in Terminal correctly. I believe the issue is that every time I try to use SelfControl I have to enter the administrator's password on my laptop to approve the changes. Has anyone else had this problem and if so what can I do? Thanks

**Comment by slambert** [comment]
What version of SelfControl are you using?

> ⚠️ Auto-SelfControl is not compatible SelfControl 4. We are trying to solve https://github.com/andreasgrill/auto-selfcontrol/issues/64, but as of today Auto-SelfControl won't be able to schedule blocks.
